discord/emoji.py

```diff
@@ -27,26 +27,20 @@
 from .asset import Asset
 from . import utils
 
-class PartialEmoji(Asset):
+class PartialEmoji:
     """Represents a "partial" emoji.
 
     This model will be given in two scenarios:
 
     - "Raw" data events such as :func:`on_raw_reaction_add`
     - Custom emoji that the bot cannot see from e.g. :attr:`Message.reactions`
 
-    This inherits from :class:`Asset`.
-
     .. container:: operations
 
         .. describe:: x == y
 
             Checks if two emoji are the same.
 
```

Comment:
Add back the `x != y` tidbit, @bmintz was right in that it's implicitly defined so there was no reason to remove it, albeit being explicit is better. When I offered the suggestion last time I meant that you should define `__ne__` not that you should remove the documentation.

Reply:
Ahh! I completely misinterpreted it. That should be easy enough to do.